bin/oneoff/remove_authorized_teacher_permissions_from_students

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env ruby
 
+time_started = Time.now
 require_relative '../../dashboard/config/environment'
+puts "Dashboard environment loaded in #{Time.now - time_started}"
```

Comment:
Did you mean to keep this in the script?  (I've noticed that this is taking a very long time now too.)